src/database/database.h

```diff
@@ -181,9 +181,7 @@ class Database {
     /// the object ID of the container that is last in the path. The
     /// updateID will hold the objectID of the container that was changed,
     /// in case new containers were created during the operation.
-    virtual void addContainerChain(std::string path, const std::string& lastClass, int flags, int lastRefID, int* containerID,
-        std::deque<int>& updateID, const std::vector<std::pair<std::string, std::string>>& lastMetadata)
-        = 0;
+    virtual bool addContainer(int parentContainerId, std::string virtualPath, const std::shared_ptr<CdsContainer>& cont, int* containerID) = 0;
```

Comment:
Is virtual path multi-level? or do we just support a single "name" now? If so, I think renaming path is sensible.

Reply:
The name is stored in `cont->title`. `virtualPath` is needed to detect existing entries and therefor multilevel.